owners/app.yaml

```diff
@@ -1,6 +1,15 @@
-runtime: nodejs
-vm: true
+# Copyright 2019, the AMP HTML authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-# Temporary setting to keep gcloud from uploading node_modules
-skip_files:
- - ^node_modules$
```

Comment:
Curious: Was `yarn` being run on the old GCE instance?

Reply:
No, `npm install` was used. GCE doesn't use `app.yaml`, so I think this was just a relic of a past attempt at a GAE migration